packages/runtime/container-runtime/src/summaryStorageCompressionHooks.ts

```diff
@@ -0,0 +1,131 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { deflate, inflate } from "pako";
+import { compress, decompress } from "lz4js";
+import { ISummaryContext } from "@fluidframework/driver-definitions";
+import { ISummaryTree, ISnapshotTree, ISummaryBlob, SummaryType } from "@fluidframework/protocol-definitions";
+import { Uint8ArrayToString } from "@fluidframework/common-utils";
+import { getBlobAtPath, listBlobPaths, replaceSummaryObject, SummaryStorageHooks } from "./summaryStorageAdapter";
+import { BlobHeaderBuilder, readBlobHeader, skipHeader, writeBlobHeader } from "./summaryBlobProtocol";
+
+export enum Algorithms {
+    NONE = 1,
+    LZ4 = 2,
+    DEFLATE = 3,
+}
+
+const ALGORITHM_KEY = "ALG";
```

Comment:
Minor style Nits: enum values are Pascal cased (None, LZ4, Deflate) and constants are camel cased like normal variables (algorithmKey).

Reply:
Implemented in f16b576ffe91c8c6593bfd311588970a03632a28